### DISCUSSION TITLE
What is the difference between Joomla Menu & Astroid Menu?

### DISCUSSION THREAD

**Discussion by paurray** [templaza/astroid-framework#359] [Q&A]
Hello Astroid Community

I am a little confused.
Since “Astroid Framework Version 2.6.6”
https://github.com/templaza/astroid-framework/releases/tag/v2.6.6
There is a new item added
“Add Astroid Menu Module”
I do not understand:
What is the difference between Joomla Menu & Astroid Menu?
Can not find anything in the documentation…

Thanks for any pointers

Paul

**Comment by sonvnn** [comment]
It displays the Joomla Menu in Astroid style. Looks like Joomla Menu and configuration are the same.

**Comment by paurray** [comment]
@sonvnn Thank you for this. Really helps.
On a side note I hope to start announcing some Astroid tutorials in the coming days.
